sdk/core/Azure.Core/tests/public/PatchModels/ChildPatchModel.Serialization.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using Azure.Core.Json;
+using Azure.Core.Serialization;
+
+namespace Azure.Core.Tests.PatchModels
+{
+    public partial class ChildPatchModel : IModelJsonSerializable<ChildPatchModel>
+    {
+        ChildPatchModel IModelJsonSerializable<ChildPatchModel>.Deserialize(ref Utf8JsonReader reader, ModelSerializerOptions options)
+        {
+            MutableJsonDocument mdoc = MutableJsonDocument.Parse(ref reader);
```

Comment:
nit: missing the [validation call](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Shared/ModelSerializerHelper.cs#L13) on each of these implementations.